docs/products/opensearch/dashboards/index.rst

```diff
@@ -0,0 +1,31 @@
+OpenSearch Dashboards
+======================
+
+OpenSearch Dashboards is both a visualisation tool for data in the cluster and a user interface for OpenSearch plugins.
```

Comment:
Can we mention that it's included with your Aiven for OpenSearch service, and accessible through the browser?